scripts/CMakeLists.txt

```diff
@@ -303,13 +303,12 @@ ELSE()
         COMPONENT ${${file}_COMPONENT}
       )
     ENDFOREACH()
-  ENDIF()
-  IF (NOT WITHOUT_SERVER)
     SET(SERVER_SCRIPTS
       mariadb-fix-extensions
       mariadbd-multi
       mariadbd-safe
       mariadb-dumpslow
+      ${SYSTEMD_SCRIPTS}
```

Comment:
Defined in `cmake/systemd.cmake` `   SET(SYSTEMD_SCRIPTS mariadb-service-convert galera_new_cluster galera_recovery)` . I think it is ok. Man pages are also related to the server.